Resources/public/views/recommendations.html.twig

```diff
@@ -0,0 +1,31 @@
+<div id="recommendation-{{ recommendationId }}" class="ez-recommendation">
+    {{ 'Loading recommended items...'|trans }}
+</div>
+
+<script id="recommendation-template-{{ recommendationId }}" type="text/x-handlebars-template">
+    {{ template|raw }}
+</script>
+
+<script>
+    (function() {
+        var recoClient{{ recommendationId }} = new EzRecoRestClient({
+            scenarioId: '{{ scenarioId }}',
+            locationId: '{{ locationId }}',
+            limit: '{{ limit }}',
+            restUrl: '/recommendations/fetch' // TODO: dynamically replace with path from routing.yml
```

Comment:
I'm sorry, but I disagree :-)

One basic rule of agile is that you don't add something until it is required. If we have a use-case where it is necessary, we can consider it, but as far as I can see, we don't.